### PR TITLE
Update quotes in spec doc

### LINKED ISSUE #100: Opening and closing quotes are inconsistent throughout the specification document
For example two code examples:

1. `<c:out value="${customer.address.city}" default="unknown"/>`
This example uses straight double quotes.

2. `<c:set var=”foo” value=”value”/>`
This example uses curly quotes

We should at the very least be consistent in our examples.

I'm open to changing this to option 1 throughout but for typographical reasons some may disagree.

Thoughts on 1/2?

https://practicaltypography.com/straight-and-curly-quotes.html

### PR DESCRIPTION
**Pull request by pnicolucci**
fixes #100

### CODE REVIEW

**Comment by pnicolucci** [comment]
@arjantijms can you give this one a close review for me? It is a bit hard on the eyes so I apologize in advance! Mainly verifying I didn't make any mistakes such as : https://github.com/eclipse-ee4j/jstl-api/pull/106/commits/7e0d9c03cd3d736b5e94fc857d48c15898fd2155 I have a list of notes that I'll work though as well as some of the examples seem to be missing necessary quotes in places. I'll work on a PR for some other minor corrections soon.

File: spec/src/main/asciidoc/jakarta-stl.adoc [hunk 1]
```diff
@@ -1030,7 +1029,7 @@ content, then the action trims it and processes it further.
 
 | _escapeXml_ |
 _true_ | _boolean_
-|Deterrmines whether characters <,>,&,’,” in
+|Deterrmines whether characters <,>,&,'," in
```
Comment (arjantijms) [inline spec/src/main/asciidoc/jakarta-stl.adoc:1032]: Minor type, not related to PR.

**Comment by arjantijms** [comment]
It looks good, couldn't find any unbalanced quoting. I did notice the quotes in the normal text changed as well, which might not have been necessary, but it's perfectly fine with me. Thx!

File: spec/src/main/asciidoc/jakarta-stl.adoc (hunk 1 above)
Comment (pnicolucci) [reply spec/src/main/asciidoc/jakarta-stl.adoc:1032]: I'll fix this typo in my next PR for minor corrections, I have a few to clean up that I've noticed while working through the doc. Good catch!

Comment (pnicolucci) [reply spec/src/main/asciidoc/jakarta-stl.adoc:1032]: Fixing here: https://github.com/eclipse-ee4j/jstl-api/pull/109

**Pull request merged**